src/main/scala/net/cakesolutions/CakePublishMavenPlugin.scala

```diff
@@ -61,20 +59,21 @@ object CakePublishMavenPlugin extends AutoPlugin {
 
     def addPublishingCredentials(): Setting[Task[Seq[Credentials]]] = {
       credentials ++= {
-        if ((Path.userHome / ".sbt" / "0.13" / ".credentials").exists()) {
-          Seq(Credentials(Path.userHome / ".sbt" / "0.13" / ".credentials"))
+        if ((Path.userHome / ".sbt" / "1.0" / ".credentials").exists()) {
```

Comment:
Can't we factor these SBT version numbers away from here? Changing the SBT version value shouldn't have a surprising change here.

Reply:
Do you mean getting the credentials directly in `~/.sbt` ?
`(Path.userHome / ".sbt" / ".credentials")`